src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -2265,6 +2266,24 @@ void MarkInterfaceImplementations (TypeDefinition type)
 			}
 		}
 
+		/// <summary>
+		/// Removes the 'override' annotation for implementation of static interface methods when the interface method is removed.
+		/// </summary>
+		void RemoveStaticInterfaceOverrideAnnotations (TypeDefinition type)
```

Comment:
Is there a way to write a test for this?
I don't know if the test infra actually verifies the "overrides" (maybe it should at least as an opt-in)

Reply:
It doesn't look like there's anything in the test infra yet, but I'll look into it more and add a `KeptOverride`/`NoOverride` attribute to check for that

Reply:
I've added a couple attribute to check for the presence/removal of the override annotations. They only perform a check if there is a `KeptOverride` or `RemovedOverride`, so it doesn't affect any other tests.